packaging/macos/buildpackage.sh

```diff
@@ -28,7 +29,7 @@ if [[ "${OSTYPE}" != "darwin"* ]]; then
 fi
 
 # Set the working dir to the location of this script
-cd "$(dirname "${0}")" || exit 1
+cd "$(dirname "${0}")"
```

Comment:
trivial: `pushd`

Reply:
shellcheck says that pushd is undefined in POSIX sh